src/index.ts

```diff
@@ -134,7 +139,14 @@ function onError(err: AxiosError) {
   const onBackoffPromise = new Promise(resolve => {
     // Calculate time to wait with exponential backoff.
     // Formula: (2^c - 1 / 2) * 1000
-    const delay = ((Math.pow(2, config.currentRetryAttempt!) - 1) / 2) * 1000;
+    let delay: number;
```

Comment:
To make this a little more clear, can we set the backoffType?

```js
config.backoffType = config.backoffType || 'expotential';
```

That way we can avoid the checking below :)

Reply:
done